### PR TITLE
changing provisioned_capacity from RU to vCPU - changes to descriptio…

### PR DESCRIPTION
**Pull request by marksoper**
Adapting Standard Tier `provisioned_capacity` to vCPU instead of RU
- changing docs annotations from RU to vCPU
- changing example values multiple of 500 to multiple of 1 - e.g. 1000 changed to 2

**Commit checklist**
- [ ] Changelog
- [ ] Doc gen (`make generate`)
- [ ] Integration test(s)
- [ ] Acceptance test(s)
- [ ] Example(s)

### CODE REVIEW

File: docs/data-sources/cluster.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Read-Only:
 
 Read-Only:
 
-- `provisioned_capacity` (Number) Maximum number of Request Units that the cluster can consume per second.
+- `provisioned_capacity` (Number) Number of estimated vCPUs available to the cluster
```
Comment (andy-kimball) [inline docs/data-sources/cluster.md:88]: NIT: Needs an ending period.

File: docs/resources/cluster.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Read-Only:
 
 Optional:
 
-- `provisioned_capacity` (Number) Maximum number of Request Units that the cluster can consume per second.
+- `provisioned_capacity` (Number) Number of estimated vCPUs available to the cluster
```
Comment (andy-kimball) [inline docs/resources/cluster.md:96]: NIT: ending period

**Review by andy-kimball — APPROVED** [review]
LGTM, with a couple nits

**Comment by fantapop** [comment]
@marksoper IIRC you were in favor of closing this right?

**Pull request closed without merging**